test/hotspot/jtreg/compiler/patches/java.base/java/lang/Helper.java

```diff
@@ -132,14 +132,6 @@ public static boolean contentEquals(byte[] value, CharSequence cs, int len) {
         return StringUTF16.contentEquals(value, cs, len);
     }
 
-    public static int putCharsAt(byte[] value, int i, char c1, char c2, char c3, char c4) {
-        return StringUTF16.putCharsAt(value, i, c1, c2, c3, c4);
```

Comment:
Why do we remove the tests for UTF16? And should we add another set of test for LATIN1 too?

Reply:
An early version removed putCharsAt, so it was also removed from Helpers. I have added it back.